web/app/js/components/util/ApiHelpers.jsx

```diff
@@ -137,14 +138,23 @@ export const ApiHelpers = (pathPrefix, defaultMetricsWindow = '10m') => {
   };
 
   // prefix all links in the app with `pathPrefix`
-  const ConduitLink = props => {
-    let {to, absolute} = props;
-
-    if (absolute) {
-      return <Link to={to} target="_blank">{props.children}</Link>;
-    } else {
-      return <Link to={`${pathPrefix}${to}`}>{props.children}</Link>;
+  class ConduitLink extends React.Component {
+    render() {
+      let prefix = pathPrefix.replace("/web:", "/"+this.props.deployment+":");
```

Comment:
thinking more about this, since most `ConduitLinks` are currently to `web`... we're replacing /web: with /web: most of the time. what if we didn't default deployment to web, and only ran `pathPrefix.replace` if `props.deployment != ""`?